modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -38,6 +38,10 @@ class Emails {
         onValidationFailure,
     void Function(Object e)? onError,
   }) async {
+    if (AuthConfig.current.validatePasswordHashCallback != null) {
+      return AuthConfig.current.validatePasswordHashCallback!
+          .call(password, hash);
+    }
     try {
       return await PasswordHash(
```

Comment:
Suggestion: Assign this using the new interface as a callback to the config directly instead of branching inside of this method.